modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -258,7 +249,44 @@ public void testGetOrganization() {
                 .body("id", equalTo(organizationID));
     }
 
-    @Test(dependsOnMethods = "testGetOrganization")
+    @DataProvider(name = "filterQueries")
+    public Object[][] createFilterQueries() {
+        return new Object[][] {
+                {"?filter=name co G&limit=10&recursive=false", false, false},
+                {"?filter=attributes.Country co S&limit=10&recursive=false", true, false},
+                {"?filter=attributes.Country eq Sri Lanka&limit=10&recursive=false", true, false},
+                {"?filter=attributes.Country eq USA&limit=10&recursive=false", false, true}
+        };
+    }
+
+    @Test(dependsOnMethods = "testGetOrganization", dataProvider = "filterQueries")
+    public void testGetOrganizations(String filterQuery, boolean expectAttributes, boolean expectEmptyList) {
```

Comment:
```suggestion
    public void testFilterOrganizations(String filterQuery, boolean expectAttributes, boolean expectEmptyList) {
```

Reply:
This is now updated to `testFilterOrganizations`